packages/devtools_app/test/analytics_prompt_test.dart

```diff
@@ -17,13 +17,13 @@ import 'package:flutter_test/flutter_test.dart';
 const windowSize = Size(2000.0, 1000.0);
 
 void main() {
-  AnalyticsController _controller;
+  AnalyticsController? _controller;
 
-  bool didCallEnableAnalytics;
+  bool? didCallEnableAnalytics;
 
   Widget _wrapWithAnalytics(
     Widget child, {
-    AnalyticsController controller,
+    AnalyticsController? controller,
   }) {
     _controller ??= controller;
```

Comment:
This is a bug. If the controller was specified, we should use it. Replace with:
```
if (controller != null) {
  _controller = controller;
}
```

Reply:
done